### ISSUE TITLE
how to use Magma foundation model to solve downstream task?

### ISSUE THREAD

**Issue by zhangxiaodi** [microsoft/Magma#29]
I want to use a pre-trained model for downstream tasks like robot manipulation. Are there any clear guidelines for this? Or do you know any useful projects on how to use LLMs or MLLMs for robot manipulation?

**Comment by jwyang** [comment]
Hi, @zhangxiaodi , maybe you can refer to the agents/libero folder. In the meanwhile, we are going to release the finetuning code on rootics data by the end of this week.

**Comment by melaslalib** [comment]
Hi, i am not professional in that field. i have question can we use the pretrained model that from streaming camera to detecte lanes ( left and right lane) from robot camera and return mask for that?

**Comment by jwyang** [comment]
Unfortunately, this model by default will not be able to return mask.

**Comment by LostXine** [comment]
> > Hi, [@zhangxiaodi](https://github.com/zhangxiaodi) , maybe you can refer to the agents/libero folder. In the meanwhile, we are going to release the finetuning code on rootics data by the end of this week.
> 
> "magma-8b-libero-goal" model seems to be missing from huggingface

+1

It would be very great if you could release checkpoints regarding LIBERO, as listed in this [README](https://github.com/microsoft/Magma/blob/main/agents/libero/README.md). 

Thank you so much for your help.
